### ISSUE TITLE
Update CogVideoX-Fun model with latest I2V model?

### ISSUE THREAD

**Issue by phr00t** [aigc-apps/CogVideoX-Fun#19]
A newer image-to-video model was released by THUDM here:

https://huggingface.co/THUDM/CogVideoX-5b-I2V

I've seen multiple reports that it provides better quality image-to-video results than the original model, which I presume CogVideoX-Fun is based on. Can we get the CogVideoX-Fun treatment for this new I2V model to allow more resolutions, final images, samplers, lengths etc. for the CogVideoX-I2V 5B model?

Thank you!

**Comment by AA-Developer** [comment]
Yes I have tested the image to video (CogVideoX-5b-I2V) model and it is very stable compared to the previous model.
It would be great to update the interface and release a version of CogVideoX-Fun  Because it has much stronger control than the original version.

**Comment by bubbliiiing** [comment]
It is the process of fine-tuning the original i2v model with CogVideoX-Fun again?

**Comment by phr00t** [comment]
> It is the process of fine-tuning the original i2v model with CogVideoX-Fun again?

Yes please!
